### PR TITLE
Resizing tiles when window size changes

### LINKED ISSUE #7: Resizing window should resize tiles
Resizing the window should resize the tiles together with the tile number. The aspect ratio should stay the same.

### PR DESCRIPTION
**Pull request by ondralukes**
Fixed #7

### CODE REVIEW

File: NumberView.cpp [hunk 1]
```diff
@@ -8,14 +8,6 @@
...
 #include <iostream>
 
-float squareSize = 100;
-
-void ScaleBy (float scale)
-{
-	squareSize *= scale;
-	// UpdateTiles();
-}
-
```
Comment (ondralukes) [inline NumberView.cpp]: This function isn't used anywhere

**Comment by pulkomandy** [comment]
It's not so nice that WindowBoard is now accessing NumberView size field directly. Why not call the ScaleBy method instead? (and make it do the right thing, if needed).

**Comment by ondralukes** [comment]
OK, I'll do it

**Comment by ondralukes** [comment]
It's better now. Thanks for help.

**Comment by humdingerb** [comment]
I tested and it does fix #7. There are other issues that I'll open tickets for in a minute.

**Pull request merged**